web/config/environment.js

```diff
@@ -34,20 +32,15 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
-    metricsAdapters: hasGoogleAnalyticsID
-      ? [
-          {
-            name: "GoogleAnalyticsFour",
-            environments: ["development", "production"],
-            config: {
-              id: getEnv("GOOGLE_ANALYTICS_TAG_ID"),
```

Comment:
Curious if setting tag ID via the `GOOGLE_ANALYTICS_TAG_ID` environment variable will be still supported?